docs/notebooks/nmf-wikipedia.ipynb

```diff
@@ -0,0 +1,3513 @@
+{
```

Comment:
1. Re-run it (with latest code version)
2. Descriptions (what & why)
3. Disable debug logs

Reply:
this file still outdated, why?